src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -8,286 +8,384 @@
+                        :disabled="isOrg"
+                        :rules="roleRules"
                     />
-                  </div>
-                </div>
-
-                <!-- Delivery address -->
-                <div v-if="isDirector || isProprietor || isPartner" class="form__row">
-                  <v-checkbox
-                    class="mt-0"
-                    label="Delivery Address same as Mailing Address"
-                    v-model="inheritMailingAddress"
-                    :disabled="disableSameDeliveryAddress"
+                    </div>
+                  </v-col>
+                  <v-col cols="4" class="mt-0" v-if="isPerson">
+                    <div class="pa-1">
+                      <v-checkbox
+                        id="dir-checkbox"
+                        class="mt-1"
+                        v-model="selectedRoles"
+                        :value="RoleTypes.DIRECTOR"
+                        :label="RoleTypes.DIRECTOR"
+                        :rules="roleRules"
+                        @change="assignDirectorRole()"
+                      />
+                    </div>
+                  </v-col>
+                </v-row>
+              </article>
+
+              <!-- Mailing address -->
+              <article class="mt-6 mb-n6">
+                <label class="sub-header">Mailing Address</label>
+                <div class="address-wrapper pt-4">
+                  <!-- NB: prevent edit when business was looked up -->
+                  <MailingAddress
+                    ref="mailingAddress"
+                    :editing="true"
+                    :schema="mailingAddressSchema"
+                    :address="inProgressMailingAddress"
+                    @update:address="updateAddress(inProgressMailingAddress, $event)"
+                    @valid="mailingAddressValid = $event"
                   />
-                  <div v-if="!inheritMailingAddress || disableSameDeliveryAddress" class="mt-4">
+                </div>
+              </article>
+
+              <!-- Delivery address (director/proprietor/partner only) -->
+              <article v-if="isDirector || isProprietor || isPartner" class="mt-6">
+                <v-checkbox
+                  class="mt-0 pt-0 mb-n2"
+                  label="Delivery Address same as Mailing Address"
+                  v-model="inheritMailingAddress"
+                  :disabled="disableSameDeliveryAddress"
+                />
+                <v-expand-transition>
+                  <div v-if="!inheritMailingAddress || disableSameDeliveryAddress" class="mt-6 mb-n6">
                     <label class="sub-header">Delivery Address</label>
                     <div class="address-wrapper pt-4">
                       <!-- NB: prevent edit when business was looked up -->
                       <DeliveryAddress
                         ref="deliveryAddress"
-                        :editing="!orgPerson.isBusinessLookup"
+                        :editing="true"
```

Comment:
As long as we show the Delivery Address, it should be editable. (Dunno what I was thinking when I originally did this.)